mkt/site/tests/__init__.py

```diff
@@ -430,6 +430,12 @@ def shortDescription(self):
     def _pre_setup(self):
         super(TestCase, self)._pre_setup()
 
+        # XXX See if we can remove this when we switch to Django 1.8.
+        # Some crud from the migration system gets stuck here and we have to
+        # kick it loose manually.
+        from mkt.webapps.models import AddonUser
+        UserProfile.addonuser_set.related.model = AddonUser
```

Comment:
But why does this bleed in our tests, and why is it only a problem for this particular FK ? This should not happen, so I feel we should figure out whether there is an actual bug on django's side or ours before proceeding with this hack